volume/store/store.go

```diff
@@ -194,12 +194,14 @@ func (s *VolumeStore) create(name, driverName string, opts map[string]string) (v
 		}
 	}
 
-	logrus.Debugf("Registering new volume reference: driver %q, name %q", driverName, name)
```

Comment:
Hum, it doesn't hurt to have the log before too, right :smile_cat: (and it's `Debug` logs so…). I'm fine with keeping it (maybe without the `driverName` if empty), Idk :sweat_smile:.